### ISSUE TITLE
Do you support F-Droid inclusion?

### ISSUE THREAD

**Issue by egils-consulting** [airgap-it/airgap-wallet#35]

**Comment by AndreasGassmann** [comment]
Yes, we would like to include our apps in the F-Droid store. This was actually a main reason why we decided to set a permissive license for the project: https://github.com/airgap-it/airgap-wallet/issues/2.

I assume there will be some issues here, one of them the usage of the google play services (for push notifications). How can this be solved, without removing the feature from the play store version of the app?

**Comment by trymeouteh** [comment]
Others apps on F-Droid do have notifications without Google play services. It is done by having a build in notification system in the app, this can cause the battery to drain quicker but you can have settings in the app on how often it will check for new transactions like every hour or two instead of constantly for F-Droid users.

**Comment by prograde** [comment]
Check out the [MicroG](https://microg.org/) project! 

> Service Core (GmsCore) is a library app, providing the functionality required to run apps that use Google Play Services or Google Maps Android API (v2).

> Services Framework Proxy (GsfProxy) is a small helper utility to allow apps developed for Google Cloud to Device Messaging (C2DM) to use the compatible Google Cloud Messaging service included with GmsCore.

**Comment by loziniak** [comment]
Seems there are some communication issues, @AndreasGassmann seems F-Droid team is waiting for your response: https://gitlab.com/fdroid/rfp/-/issues/1479

**Comment by martin-braun** [comment]
An alternative is to use the [web version](https://wallet.airgap.it). The Vault is available on F-Droid.
